### PR TITLE
feat: Add support for WPF OpenGL acceleration

### PR DESCRIPTION
**Pull request by jeromelaban**
GitHub Issue (If applicable): https://github.com/unoplatform/uno/issues/4309, fixes https://github.com/unoplatform/uno/issues/10461

## PR Type

What kind of change does this PR introduce?
- Feature

## What is the new behavior?

Enable OpenGL acceleration for WPF head.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.Runtime.Skia.Wpf/WpfHost.cs [hunk 1]
```diff
@@ -109,6 +109,12 @@ internal static void RegisterExtensions()
 
 		public static WpfHost? Current => _current;
 
+		/// <summary>
+		/// Gets or sets the current Skia Render surface type.
+		/// </summary>
+		/// <remarks>If <c>null</c>, the host will try to determine the most compatible mode.</remarks>
+		public RenderSurfaceType? RenderSurfaceType { get; set; }
```
Comment (Youssef1313) [inline src/Uno.UI.Runtime.Skia.Wpf/WpfHost.cs:116]: Should this be made non-nullable with OpenGL being the default value?

**Comment by FrozDark** [comment]
Does it solve https://github.com/unoplatform/uno/issues/10461 ?

File: src/Uno.UI.Runtime.Skia.Wpf/WpfHost.cs (hunk 1 above)
Comment (jeromelaban) [reply src/Uno.UI.Runtime.Skia.Wpf/WpfHost.cs:116]: The idea is to keep consistency with the GTK implementation which uses null for auto detection, as we may add more backends later.

**Comment by jeromelaban** [comment]
@FrozDark it will for WPF, yes. You can try with the packages from https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=70008&view=results, or wait for the preview package.

**Review by dansiegel — APPROVED** [review]

**Pull request merged**